core/src/main/java/discord4j/core/event/domain/message/ReactionUserEmojiEvent.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * This file is part of Discord4J.
+ *
+ * Discord4J is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Discord4J is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Discord4J. If not, see <http://www.gnu.org/licenses/>.
+ */
+package discord4j.core.event.domain.message;
+
+import discord4j.common.util.Snowflake;
+import discord4j.core.GatewayDiscordClient;
+import discord4j.core.object.emoji.Emoji;
+import discord4j.core.object.entity.User;
+import discord4j.core.object.reaction.Reaction;
+import discord4j.gateway.ShardInfo;
+import reactor.core.publisher.Mono;
+import reactor.util.annotation.Nullable;
+
+/**
+ * Represents an event related to a {@link discord4j.core.object.reaction.Reaction} with a user and more data of the reaction.
+ */
+public abstract class ReactionUserEmojiEvent extends ReactionBaseEmojiEvent {
```

Comment:
not a fan for this name... 
any suggestion?

Reply:
`UserTriggeredEmojiReactionEvent extends AbstractEmojiReactionEvent`

Reply:
Feels strange not start with `
`Reaction
And i wanna avoid the Abstract word based in how the MessageEvent dont use that